dubbo-common/src/main/java/org/apache/dubbo/config/AbstractInterfaceConfig.java

```diff
@@ -748,7 +748,7 @@
         if (methodsJson != null) {
             this.methods = new ArrayList<>();
             methodsJson.forEach(
-                    (methodConfigJson) -> methods.add(JsonUtils.toJavaObject(methodConfigJson, MethodConfig.class)));
+                    methodConfigJson -> methods.add(JsonUtils.toJavaObject(methodConfigJson, MethodConfig.class)));
```

Comment:
**Suggestion**: Removing unnecessary parentheses in lambda expressions can simplify the code. Good job on this cleanup. <br> **Code Suggestion**: 
 ```
 The provided change is correctly applied as suggested.
 ```